addons/account/views/report_invoice.xml

```diff
@@ -23,7 +23,10 @@
 
                     <div id="informations" class="row mt32 mb32">
                         <div class="col-auto mw-100 mb-2" t-if="o.invoice_date" name="invoice_date">
-                            <strong>Invoice Date:</strong>
+                            <span t-if="o.type == 'out_invoice'"><strong>Invoice Date:</strong></span>
```

Comment:
Adding a span can change the CSS rendering

Reply:
without span, (moving the `t-if` in `<strong>`)  has a side effect that it wasn't export in .pot/.po files, I don't know why

Reply:
But you are not updating the `.pot` file. If you don't do it, the translators won't be able to do their job

Reply:
Thanks @william-andre I thought there was some automatic handler to manage translations to avoid conflict while forwarding patch, last commit add change on .pot file regarding this.

Reply:
It is done automatically only in `master`, when freezing the branch to a new stable version

Reply:
> without span, (moving the `t-if` in `\<strong\>`) has a side effect that it wasn't export in .pot/.po files, I don't know why

Is it not working with a `<t/>` tag instead of a `<span/>`?

Reply:
your're right much better ! done with the last commit !